.github/workflows/update_dependencies.yml

```diff
@@ -8,33 +8,48 @@ jobs:
   update-dependencies:
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: "${{ secrets.NIGHTLY_TESTING }}"
+
+      - name: Get PR and labels
```

Comment:
I copied this step from our main build.yml.in file, where it's used to check if "auto-merge-after-CI" is present.